proxy/grpcproxy/cluster.go

```diff
@@ -25,6 +25,7 @@ import (
 	"go.etcd.io/etcd/v3/etcdserver/api/v3rpc/rpctypes"
 	pb "go.etcd.io/etcd/v3/etcdserver/etcdserverpb"
 
+	"errors"
```

Comment:
this import needs to be fixed. Put it together with the std lib import block.

Reply:
Fixed.